app/javascript/Search/Search.jsx

```diff
@@ -81,8 +106,9 @@ export class Search extends Component {
     }
   }
 
-  componentDidUnmount() {
+  componentWillUnmount() {
```

Comment:
The wrong [mount lifecycle method](https://reactjs.org/docs/react-component.html#mounting) was being used. `componentDidUnmount` no longer exists. This was never caught after the Preact upgraded.